components/bl-list-component/styles/index.less

```diff
@@ -0,0 +1,19 @@
+@bl-customComponent-list-theme: @themePrimary;
+@bl-customComponent-list-themeTextColor: @appTextColor;
+
+@bl-customComponent-list-item-fontSize: 1rem;
+
+.bl-customComponent-list {
+  color: @bl-customComponent-list-themeTextColor;
+  user-select: none;
```

Comment:
variable for that

Also, I do not quite understand why we can't user-select for list content. It's disabling copying text from the list, which is pretty annoying for me personally.

I believe we could do remove this line, or leave with default value and create variable for that(which is better). Or - we could create an property for that.